app/facades/NWS_facade.rb

```diff
@@ -9,4 +9,14 @@ def self.retrieve_disaster(location)
             end
         end
     end
+
+    def self.retrieve_disaster_sms(location, phone_number)
+        weather_events = NWSService.get_disaster(location)
+        if weather_events[:features].empty? == false 
```

Comment:
Could use .any? for this. `weather_events[:features].any?` would return true if the array is populated with anything so you wouldn't need to extra comparison.